assets/js/lesson_5/hw_10.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * I will give names for variables like taskOne, taskTwo , etc.
+ * and add description to every new task
+ */
+
+// 1.Show numbers from 10 to 20 in one line
+
+let taskOne = "";
+
+for (let i = 10; i <= 20; i++) {
+    taskOne += i + ",";
+}
+
+console.log(taskOne.replace(/,\s*$/, ""));
```

Comment:
it will be easier to push all data to array and then use `array.join(' ')` to show result

Reply:
I see variation